rai/show.go

```diff
@@ -127,3 +127,42 @@ func (tx *TransactionResult) Show() {
 		}
 	}
 }
+
+func zip(lists ...[]interface{}) func() []interface{} {
+	zip := make([]interface{}, len(lists))
+	i := 0
+	return func() []interface{} {
+		for j := range lists {
+			if i >= len(lists[j]) {
+				return nil
+			}
+			zip[j] = lists[j][i]
+		}
+		i++
+		return zip
+	}
+}
+
+func (tx *TransactionAsyncResult) Show() {
+	res := make(map[string][][]interface{})
+
+	for _, r := range tx.Results {
+		res[r.RelationID] = append(res[r.RelationID], r.Table)
+	}
+
+	for k, v := range res {
```

Comment:
It looks like you were building this intermediate map in case there were multiple Tables that have the same relationId.

But that should never happen, I think. I think every relation in tx.Results should have a unique RelationId.

So then, I think this might be simpler?:
```suggestion
	for _, r := range tx.Results {
		k = r.RelationID
		v = r.Table
```
And then maybe you can just `zip()` the columns for each table, I think.

Reply:
(it's gonna need some work besides this though, in the code below it)